doozer/doozerlib/cli/images_streams.py

```diff
@@ -970,7 +967,23 @@ def search_issues(query):
             issue = jira_client.create_issue(
                 fields,
             )
-            # check depend issues and set depend to a higher version issue if ture
+            # Evaluate the complex expression ONCE
```

Comment:
What complex expression? Why once? It gets evaluated for each component, right? Maybe drop the.. confusing comment.